packages/wdio-cli/src/utils.ts

```diff
@@ -358,6 +358,8 @@ export function hasBabelConfig(rootDir: string) {
  * detect if project has a compiler file
  */
 export async function detectCompiler(answers: Questionnair) {
+    if (answers.createPackageJSON) { return false }
```

Comment:
Are we sure we want to return `false` here? What if ther user wants to use TS in a new project?

Reply:
@christian-bromann this function is only used for __computing default answer__ to the question: "Do you want to use Typescript to write tests?"

https://github.com/webdriverio/webdriverio/blob/2f9d09151cfb3666c8fe9178326f9f943107ec33/packages/wdio-cli/src/constants.ts#L472-L483

So, by the time `detectCompiler()` is called, there is no answer for `isUsingTypeScript` yet, because `detectCompiler()` gets called right before asking user for `isUsingTypeScript`.

Reply:
Can we add a comment above the if statement to document this reasoning?